wallpaperdownloader/README.md

```diff
@@ -0,0 +1,17 @@
+# WallpaperDownloader snap
+
+This project creates a working snap of WallpaperDownloader application.
```

Comment:
Something grammatical ;)

`This project creates a working snap of the WallpaperDownloader application.`